### ISSUE TITLE
Execution error (ClassCastException) in java.clj

### ISSUE THREAD

**Issue by scarytom** [liquidz/antq#174]
I am getting the following intermittent failure:

```sh
Execution error (ClassCastException) at antq.diff.java/eval13264$fn$fn (java.clj:87).
class clojure.lang.ExceptionInfo cannot be cast to class java.lang.CharSequence (clojure.lang.ExceptionInfo is in unnamed module of loader 'app'; java.lang.CharSequence is in module java.base of loader 'bootstrap')
```

I am running antq through deps.edn with the following alias definition:
```clj
:outdated {:replace-deps {}
           :extra-deps   {com.github.liquidz/antq {:mvn/version "1.9.863"}
                          org.slf4j/slf4j-simple  {:mvn/version "1.7.36"}}
           :main-opts    ["-m" "antq.core" "--upgrade" "--force"]}
:eastwood {:extra-paths ["src/test/clojure" "src/dev/clojure"]
```

I'm sorry that I don't have a reproducible test case for this yet, but I will update the issue if I discover anything more.

**Comment by liquidz** [comment]
@scarytom Sorry for late reply.
I've reproduced the problem with `ANTQ_MAVEN_TIMEOUT=1`.
cf. https://github.com/liquidz/antq/blob/main/doc/timeout.adoc

As you said `latest-version` had a timed out exception, so I fixed it in [dev](https://github.com/liquidz/antq/tree/dev) branch.
Could you try?

**Comment by scarytom** [comment]
I have tried this out and it appear to fix the issue, thanks.

**Comment by scarytom** [comment]
I just bought you a coffee on ko-fi.com, hope you enjoy it.

**Comment by liquidz** [comment]
@scarytom Thank you so much!
Just released [v1.9.867](https://github.com/liquidz/antq/releases/tag/1.9.867).
